cmd/snap/cmd_aliases_test.go

```diff
@@ -40,14 +40,8 @@ $ snap aliases <snap>
 Lists only the aliases defined by the specified snap.
 
 An alias noted as undefined means it was explicitly enabled or disabled but is
-not defined in the current revision of the snap; possibly temporarely (e.g
-because of a revert), if not this can be cleared with snap alias --reset.
-
-Application Options:
-      --version     Print the version and exit
-
-Help Options:
-  -h, --help        Show this help message
+not defined in the current revision of the snap; possibly temporarily (e.g.
+because of a revert), if not this can be cleared with 'snap alias --reset'.
```

Comment:
I may have misunderstood something, but this reads a bit easier for me: `possibly temporarily (...), if not, ...`. Then again, I'm not a native speaker.

Reply:
yeah, a lot of these are awkward, I'll go over them. Your suggestions are good.